frontend/src/views/Layout/DashboardLayout_gdd.vue

```diff
@@ -72,12 +73,19 @@ export default {
     // DashboardContent,
     FadeTransition,
   },
+  mixins: [VueTimers],
   data() {
     return {
       balance: 0,
       GdtBalance: 0,
     }
   },
+  computed: {
+    displayedBalance() {
+      const str = String(this.balance)
+      return Number(str.substring(0, str.indexOf('.') + 3))
```

Comment:
Is this enough to have always two digits? Can this fail in case of `str = 1.1`?

Reply:
Sometimes we just get a full integer, this might change with the float change. So you might wanna do this:
`floor(this.balance,2)` or `floor(this.balance << 2) >> 2`